diff --git a/node_modules/@formatjs/cli/bin/formatjs b/node_modules/@formatjs/cli/bin/formatjs
index 801d42b..616ed70 100755
--- a/node_modules/@formatjs/cli/bin/formatjs
+++ b/node_modules/@formatjs/cli/bin/formatjs
@@ -12788,7 +12788,7 @@ var require_manipulator = __commonJS({
       }
       ast.forEach(function(el) {
         if ((0, types_1.isArgumentElement)(el) || (0, types_1.isDateElement)(el) || (0, types_1.isTimeElement)(el) || (0, types_1.isNumberElement)(el)) {
-          if (el.value in vars && vars.get(el.value) !== el.type) {
+          if (vars.has(el.value) && vars.get(el.value) !== el.type) {
             throw new Error("Variable ".concat(el.value, " has conflicting types"));
           }
           vars.set(el.value, el.type);
